.github/workflows/build_linux_packages.yml

```diff
@@ -115,7 +115,7 @@ jobs:
           aws-region: us-east-2
           role-to-assume: arn:aws:iam::692859939525:role/therock-artifacts
 
-      # TODO: Move to script
+      # TODO (#541): Move to script
```

Comment:
I think #541 should be in the PR description but it is not the appropriate reference here.

Reply:
Yeah. `Progress on https://github.com/ROCm/TheRock/issues/541`, or `Fixes https://github.com/ROCm/TheRock/issues/541` in the PR description.

https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue